org-formation/060-cloudtrail/cloudtrail-trail.yaml

```diff
@@ -122,28 +121,32 @@ Resources:
     Properties:
       Bucket: !Ref CloudTrailBucket
       PolicyDocument:
+        # policy copied from manual setup of cloudtrail with encryption
         Version: 2012-10-17
         Statement:
-          - Sid: AWSCloudTrailAclCheck
-            Effect: Allow
-            Principal:
-              Service: cloudtrail.amazonaws.com
-            Action: s3:GetBucketAcl
-            Resource: !GetAtt CloudTrailBucket.Arn
-          - Sid: AWSCloudTrailWrite
-            Effect: Allow
-            Principal:
-              Service: cloudtrail.amazonaws.com
-            Action: s3:PutObject
-            Resource: !Sub '${CloudTrailBucket.Arn}/AWSLogs/*'
-            Condition:
-              StringEquals:
-                s3:x-amz-acl: bucket-owner-full-control
+        - Sid: AWSCloudTrailAclCheck20150319
+          Effect: Allow
+          Principal:
+            Service: cloudtrail.amazonaws.com
+          Action: s3:GetBucketAcl
+          Resource: !GetAtt CloudTrailBucket.Arn
+          Condition:
```

Comment:
So that ensure that only the trail can do getBucketAcl (and putObject) on this bucket?

Reply:
yes, i believe so @xschildw